src/lib/storage/dictionary_column.hpp

```diff
@@ -92,6 +92,19 @@ class DictionaryColumn : public BaseColumn {
     _dictionary->shrink_to_fit();
   }
 
+  void _assign_attribute_vector(size_t size) {
+    Assert(size < UINT64_MAX, "Number of attributes out of range");
+    if (size < UINT8_MAX) {
```

Comment:
Das ist eine Konstante - genauer ein Makro - aus `cstdint`. Ich weiß nicht genau, wie diese überhaupt in den Scope kommt - der zugehörige Header wird zumindest nicht direkt inkludiert. Wie der Name schon sagt kommt das eigentlich aus C.  Vielleicht ist die Variante aus der C++ Standardbibliothek `std::numeric_limits<uint64_t>::max()` (Template-Methode aus dem `std`-Namespace) vorzuziehen.